src/display/api.js

```diff
@@ -1062,10 +1076,12 @@ class PDFPageProxy {
    */
   getAnnotations({ intent = null } = {}) {
     if (!this.annotationsPromise || this.annotationsIntent !== intent) {
+      /** @private */
```

Comment:
I would also think that these two are private, but they don't have an underscore. That's actually the case for more properties in this file I'm afraid, but that's a concern for another time.

Reply:
> Until the time we do an API change, these are probably regarded as public now.

Not really, as far as I'm concerned, since those properties shouldn't be considered public and/or relied upon in any way; they are only intended to be *internal* functionality.
Generally speaking, I'd say that things which are not documented in `src/display/api.js` should *not* be considered public and can thus stop working at any time.

Reply:
Anyway, I don't think that we should accept this patch since it introduces an unfortunate precedent (as far as I'm concerned).
Given the established pattern of `leading underscore == private` in JavaScript, I think it's (somewhat) up to API users to be mindful of that.

---

If we want to make any changes, simply adding leading underscores to the two properties in question here seems straightforward enough; and I'd *not* even consider it an `api-minor` change.